# Define the request
# string command      # "open_inner_gripper", "open_outer_gripper"
bool open_outer_gripper_fully
bool close_outer_gripper_fully
bool open_inner_gripper_fully
bool close_inner_gripper_fully

bool stop

uint32 linear_motor_position
uint32 outer_gripper_move_to
uint32 inner_gripper_move_to

#uint32 outer_gripper_move_to
# float64 opening_width_inner_gripper   # Not implemented yet
# float64 opening_width_outer_gripper   # Not implemented yet
---
# Define the result
bool success
